device/virtual/x86_64-kvm_x86_64-r0/README.md

```diff
@@ -0,0 +1,70 @@
+# Changing the virtual device
+
+You can control the hw sku and default factory configuration for the VS image 
+by modifying the content of the file default_sku. 
+
+The format of default_sku is a single line:
+
+```
+<hw_key> <default_preset>
+```
+
+## Allowable values for hw_key
+
+| hw_key | Device |
+| ------ | ------ |
+| Force10-S6000 | Dell Force10 S6000|
+| brcm_gearbox_vs | Similar to Force10-S6000, but implements a virtual BRCM81724 Gearbox Phy |
+
+## Allowable values for default_preset
+
+These include "t1", "l2", and "empty". See the file 
+sonic-buildimage/src/sonic-config-engine/config_samples.py for details on how 
+each default_preset value is interpreted.
+
+# Device Specific Documentation
+
+For general info on building, see https://github.com/Azure/sonic-buildimage/blob/master/README.md
+
+# Force-10-S6000
+
+This is the default VS for SONiC. To enable, set contents of default_sku to:
+
+```
+Force10-S6000 t1
+```
+
+To build:
+
+```
+make init
+make configure PLATFORM=vs
+make target/sonic-vs.img.gz
+```
+
+# brcm_gearbox_vs
+
+This sku simulates a device with a Broadcom BRCM81724 gearbox PHY. To enable,
+set default_sku to:
+ 
+
+```
+brcm_gearbox_vs t1
+```
+
+To build (same as Force-10-S6000):
+
+```
+make init
+make configure PLATFORM=vs
+make target/sonic-vs.img.gz
```

Comment:
do we need to rebuilt the image after we change the default sku?

for example, let's say if this pr is merged. can I just download the pre-build sonic-vs.img.gz and change default_sku to brcm_gearbox_vs and do config reload? or reboot? how can I simulate the gearbox?

Reply:
It appears so. I changed this file on the switch to S6000, hard powered down. My change to the default_sku file persisted, yet it was still gearbox device, and behaved like one. I can't offer any more help, if you have a suggestion something to try I can, but perhaps this is a defect for someone else to look at. For now, rebuild is how this is known to work.

Reply:
I found a way. I'll document and update.

Reply:
@lguohan docs updated, check it out.